Updating readme for sdk (#21)
litmuschaos/litmus-python: contribute/developer-guide/README.md

        'experiments/sample_category/sample_exec_chaos',
        'experiments/sample_category/sample_exec_chaos/experiment',
      ```

>     - Add `&` operator at the end of kill commands `CHAOS_KILL_COMMAND` example: `kill $(find /proc -name exe -lname '*/md5sum' 2>&1 | grep -v 'Permission denied' | awk -F/ '{print $(NF-1)}') &`.

Reviewer: ```suggestion
    - Add `&` operator in the end of kill command `CHAOS_KILL_COMMAND` example: `kill $(find /proc -name exe -lname '*/md5sum' 2>&1 | grep -v 'Permission denied' | awk -F/ '{print $(NF-1)}') &`.
```

>     - Add `&` operator at the end of kill commands `CHAOS_KILL_COMMAND` example: `kill $(find /proc -name exe -lname '*/md5sum' 2>&1 | grep -v 'Permission denied' | awk -F/ '{print $(NF-1)}') &`.
>       Because we are running this command as a background process using multi-threading.  

Reviewer: ```suggestion
    - Add `&` operator in the end of kill command `CHAOS_KILL_COMMAND` example: `kill $(find /proc -name exe -lname '*/md5sum' 2>&1 | grep -v 'Permission denied' | awk -F/ '{print $(NF-1)}') &`. As we are running chaos command as a background process in separate thread.  
```
